src/main/java/ch/njol/skript/events/EvtItem.java

```diff
@@ -39,14 +40,14 @@
 import ch.njol.skript.lang.Literal;
 import ch.njol.skript.lang.SkriptEvent;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
-import ch.njol.skript.lang.util.SimpleEvent;
 import ch.njol.util.Checker;
 
-@SuppressWarnings("deprecation")
+@SuppressWarnings({"deprecation", "unchecked"})
```

Comment:
Can't you suppress the unchecked warning on method level?

Reply:
Good call. Will change this right away!